DOC: move third-party packages to new page (#20026)
matplotlib/matplotlib: doc/devel/documenting_mpl.rst

---------------------------

Image files can directly included in pages with the ``image::`` directive.

> e.g., :file:`tutorials/intermediate/constrainedlayout_guide.py` displays 
> a couple of images as static images::

Reviewer: ```suggestion
e.g., :file:`tutorials/intermediate/constrainedlayout_guide.py`  displays examples of the feature as static images::
```
images as images sounds weirds

> e.g., :file:`tutorials/intermediate/constrainedlayout_guide.py` displays 
> a couple of images as static images::

Author: ```suggestion
e.g., :file:`tutorials/intermediate/constrainedlayout_guide.py` displays 
a couple for static images::
```
